content/pt-br/docs/tasks/access-application-cluster/access-cluster-services.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Acessando serviços em execução em clusters
+content_type: task
+weight: 140
+---
+
+<!-- overview -->
+Esta página mostra como se conectar a serviços em execução no cluster Kubernetes.
```

Comment:
Sounds better to me "se conectar aos serviços em execução" instead of "se conectar a serviços em execução". Think about it.